### YamlMime:ManagedReference
items:
- uid: System.Data.SqlTypes.SqlAlreadyFilledException
  id: SqlAlreadyFilledException
  children:
  - System.Data.SqlTypes.SqlAlreadyFilledException.#ctor
  - System.Data.SqlTypes.SqlAlreadyFilledException.#ctor(System.String)
  - System.Data.SqlTypes.SqlAlreadyFilledException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: SqlAlreadyFilledException
  nameWithType: SqlAlreadyFilledException
  fullName: System.Data.SqlTypes.SqlAlreadyFilledException
  type: Class
  summary: "A SqlAlreadyFilledException osztály célja nem egy különálló összetevő, hanem egy osztály, amelyen más osztályok származnia szabványos funkciókat használja."
  remarks: "A SqlAlreadyFilledException osztály célja nem egy különálló összetevő, hanem egy osztály, amelyen más osztályok származnia szabványos funkciókat használja."
  syntax:
    content: 'public sealed class SqlAlreadyFilledException : System.Data.SqlTypes.SqlTypeException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.Data.SqlTypes.SqlTypeException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor
  id: '#ctor'
  parent: System.Data.SqlTypes.SqlAlreadyFilledException
  langs:
  - csharp
  name: SqlAlreadyFilledException()
  nameWithType: SqlAlreadyFilledException.SqlAlreadyFilledException()
  fullName: System.Data.SqlTypes.SqlAlreadyFilledException.SqlAlreadyFilledException()
  type: Constructor
  assemblies:
  - System.Data
  namespace: System.Data.SqlTypes
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.SqlTypes.SqlAlreadyFilledException&quot;> </xref> osztály."
  remarks: "A <xref:System.Data.SqlTypes.SqlAlreadyFilledException>osztály nem készült, önálló összetevőjeként, de amelyből más osztályok származnia szabványos funkciókat osztályként.</xref:System.Data.SqlTypes.SqlAlreadyFilledException>"
  syntax:
    content: public SqlAlreadyFilledException ();
    parameters: []
  overload: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Data.SqlTypes.SqlAlreadyFilledException
  langs:
  - csharp
  name: SqlAlreadyFilledException(String)
  nameWithType: SqlAlreadyFilledException.SqlAlreadyFilledException(String)
  fullName: System.Data.SqlTypes.SqlAlreadyFilledException.SqlAlreadyFilledException(String)
  type: Constructor
  assemblies:
  - System.Data
  namespace: System.Data.SqlTypes
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.SqlTypes.SqlAlreadyFilledException&quot;> </xref> osztály."
  remarks: "A <xref:System.Data.SqlTypes.SqlAlreadyFilledException>osztály nem készült, önálló összetevőjeként, de amelyből más osztályok származnia szabványos funkciókat osztályként.</xref:System.Data.SqlTypes.SqlAlreadyFilledException>"
  syntax:
    content: public SqlAlreadyFilledException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A karakterlánc jelenítsen meg, ha a kivétel történt."
  overload: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Data.SqlTypes.SqlAlreadyFilledException
  langs:
  - csharp
  name: SqlAlreadyFilledException(String,Exception)
  nameWithType: SqlAlreadyFilledException.SqlAlreadyFilledException(String,Exception)
  fullName: System.Data.SqlTypes.SqlAlreadyFilledException.SqlAlreadyFilledException(String,Exception)
  type: Constructor
  assemblies:
  - System.Data
  namespace: System.Data.SqlTypes
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.SqlTypes.SqlAlreadyFilledException&quot;> </xref> osztály."
  remarks: "A <xref:System.Data.SqlTypes.SqlAlreadyFilledException>osztály nem készült, önálló összetevőjeként, de amelyből más osztályok származnia szabványos funkciókat osztályként.</xref:System.Data.SqlTypes.SqlAlreadyFilledException>"
  syntax:
    content: public SqlAlreadyFilledException (string message, Exception e);
    parameters:
    - id: message
      type: System.String
      description: "A karakterlánc jelenítsen meg, ha a kivétel történt."
    - id: e
      type: System.Exception
      description: "A hivatkozás belső kivételt."
  overload: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Data.SqlTypes.SqlTypeException
  isExternal: false
  name: System.Data.SqlTypes.SqlTypeException
- uid: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor
  parent: System.Data.SqlTypes.SqlAlreadyFilledException
  isExternal: false
  name: SqlAlreadyFilledException()
  nameWithType: SqlAlreadyFilledException.SqlAlreadyFilledException()
  fullName: System.Data.SqlTypes.SqlAlreadyFilledException.SqlAlreadyFilledException()
- uid: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor(System.String)
  parent: System.Data.SqlTypes.SqlAlreadyFilledException
  isExternal: false
  name: SqlAlreadyFilledException(String)
  nameWithType: SqlAlreadyFilledException.SqlAlreadyFilledException(String)
  fullName: System.Data.SqlTypes.SqlAlreadyFilledException.SqlAlreadyFilledException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor(System.String,System.Exception)
  parent: System.Data.SqlTypes.SqlAlreadyFilledException
  isExternal: false
  name: SqlAlreadyFilledException(String,Exception)
  nameWithType: SqlAlreadyFilledException.SqlAlreadyFilledException(String,Exception)
  fullName: System.Data.SqlTypes.SqlAlreadyFilledException.SqlAlreadyFilledException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Data.SqlTypes.SqlAlreadyFilledException.#ctor*
  parent: System.Data.SqlTypes.SqlAlreadyFilledException
  isExternal: false
  name: SqlAlreadyFilledException
  nameWithType: SqlAlreadyFilledException.SqlAlreadyFilledException
